lib/random_data.rb

```diff
@@ -1,23 +1,23 @@
 module RandomData
```

Comment:
Hey @jeffminnear - Bloc foundation gave some questionable advice around RandomData.

When doing `include RandomData` at the top of a file, it adds RandomData`s methods to the *global* scope. Its usually a good idea to avoid putting things in global scope as it can cause unexpected naming collisions.

To fix this:

- Make all `RandomData` methods *module* methods. Each method in `RandomData` will get a `self.` prefix. E.g. `def random_foo` signature changes to `def self.random_foo`
- Remove all `include RandomData` statements (I think its in 4 files currently in this project)
- Nothing else needed. Calls to `RandomData.random_foo` will continue to work.